gcsio/pom.xml

```diff
@@ -180,6 +180,10 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.easymock</groupId>
+      <artifactId>easymock</artifactId>
+    </dependency>
```

Comment:
We do not use `easymock` in Dataproc - this is google3 policy that we adhere in our OSS projects too.

Reply:
Removed. Got added as part of some experiment.